Tests/Testo/Invoice/QuoteAmount/QuoteAmountServiceTest.php

```
@@ -0,0 +1,373 @@
+ */
+#[Test]
+final class QuoteAmountServiceTest
+{
+    public function initializeQuoteAmountSetsDefaultsAndSaves(): void
+    {
+        $model = new QuoteAmount();
+
+        /** @var QuoteAmountRepository&m\MockInterface $repo */
+        $repo = m::mock(QuoteAmountRepository::class);
+        /** @var \Mockery\Expectation $e */
+        $e = $repo->expects('save');
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $e. Configured minimum length is 3. (undefined)

(ShortVariable)